DDCAD/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+#==========================================================================
+#  AIDA Detector description implementation 
+#--------------------------------------------------------------------------
+# Copyright (C) Organisation europeenne pour la Recherche nucleaire (CERN)
+# All rights reserved.
+#
+# For the licensing terms see $DD4hepINSTALL/LICENSE.
+# For the list of contributors see $DD4hepINSTALL/doc/CREDITS.
+#
+#==========================================================================
+SET_PROPERTY(DIRECTORY . PROPERTY PACKAGE_NAME DDCAD)
+#
+if(${ROOT_VERSION} VERSION_LESS 6.21.01)
+  return()
+endif()
+#
+# Open Asset Importer Library
+#
+find_package(assimp QUIET)
+if(TARGET assimp::assimp)
+    MESSAGE(STATUS "+++> Use existing assimp implementation")    
+else()
```

Comment:
I think we should add a flag to enable or disable the automatic building of assimp, and disable this by default.
If neither assimp is found, nor the flag is enabled we just don't build DDCAD.